### PR TITLE
Docs change

### PR DESCRIPTION
**Pull request by namebrandon**
Updated guide_advanced to include tick (quote) marks around learning_rate. Code was throwing errors without the quote marks. Did verify that using the quote marks actually implemented a search around the learning rate.

### CODE REVIEW

**Comment by alexjc** [comment]
Looks OK, one file checked in by mistake though... You should set your git to ignore .bak files, or review changes before committing in SourceTree!

**Comment by namebrandon** [comment]
Ah, my apologies. I'm a bit new to git. I will make sure it's set to ignore .bak files moving forward. Do you need to me to withdraw this and re-submit?

**Comment by alexjc** [comment]
I have a minor preference for making a new (clean) request, but you can also remove the file and push, then it should update the PR automatically.

**Comment by namebrandon** [comment]
Sure.. I'll open another one. it'll give me a chance to figure out how to fix something like this on my end.. Thanks!
